qemu/tests/live_snapshot_basic.py

```diff
@@ -63,20 +59,18 @@ def create_snapshot(self):
         """
         Create a live disk snapshot.
         """
+        params = self.parser_test_args()
+        self.snapshot_mode = params.get("snapshot_mode")
+        logging.info("Creating snapshot")
```

Comment:
Suggestion: Put this log after line 69, it might confuse users if 'creating snapshot' showing up before 'creating image' , actually we do create image first, snapshot are not creating this moment. Thanks.